src/ripple/app/ledger/impl/LedgerConsensusImp.cpp

```diff
@@ -1919,14 +1919,7 @@ void LedgerConsensusImp::addLoad(STValidation::ref val)
         app_.getFeeTrack().getLocalFee(),
         app_.getFeeTrack().getClusterFee());
 
-    std::uint32_t loadBase = app_.getFeeTrack().getLoadBase();
-
-    // Hard code the minimum fee for now:
-    std::uint32_t const minFee = 10000;
-    std::uint32_t const refFee = 10;
-    fee = std::max(fee, (minFee * loadBase + refFee - 1) / refFee);
-
-    if (fee > loadBase)
+    if (fee > app_.getFeeTrack().getLoadBase())
         val->setFieldU32(sfLoadFee, fee);
```

Comment:
I think yes. Without the manual bump, the fee is now back to it's normal lower value, which is unlikely to be larger than the load base, particularly during unit tests.